docker-compose.yml

```diff
@@ -23,6 +23,18 @@ services:
     ## Only MacOSX users need --watch-poll
     command: server --dev --watch-poll
 
+  securityc:
+    image: bpanel/securityc
+    restart: on-failure
+    ports:
+      - "80:80"
+      - "443:443"
+    env_file:
+      - ./securityc.env
+    volumes:
+      # mount local cert directory into container
+      - ./certs:/etc/ssl/nginx
```

Comment:
This seems a little unnecessary.

Reply:
Good call, now its mounted into a docker volume 👍

Reply:
Would this be necessary again if we were in an environment where we wanted to persist the certs? Can you persist docker volumes on your local disk?

Reply:
The docker volume will persist on disk between bringing up and down the containers, `docker-compose down` doesn't destroy it